adopt-a-dog/contact.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="zxx" dir="ltr">
+
+<head>
+    <meta charset="utf-8">
+    <title>Contact</title>
+</head>
+
+<body>
+    <section id="header">
```

Comment:
Avoid use of header for now. A class would be preferred. Also, the header element would be a good use. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header